hg2git.py

```diff
@@ -35,7 +35,9 @@ def setup_repo(url):
   except TypeError:
     myui=ui.ui()
     myui.setconfig('ui', 'interactive', 'off')
-  return myui,hg.repository(myui,url)
+    # Avoids a warning when the repository has obsolete markers
+    myui.setconfig('experimental', 'evolution.createmarkers', True)
+  return myui,hg.repository(myui,url).unfiltered()
```

Comment:
Do this in a separate commit as it is not strictly required for the main functionality.

Reply:
I don't use pull requests much, is the best way to do this to just close this request and open another two, one for each part?